#!/usr/bin/env python
PACKAGE = "no_entry_layer"
import math
math.pi
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("mask_on",    bool_t,    0, "mask on", False)
gen.add("cost_value",    int_t,    0, "value of cost", 0,  0, 255)
gen.add("mask_lb_x",    int_t,    0, "bottom left x", 0,  0, 3671)
gen.add("mask_lb_y",    int_t,    0, "bottom left y", 0,  0, 5610)
gen.add("mask_rt_x",    int_t,    0, "top right x", 0,  0, 3671)
gen.add("mask_rt_y",    int_t,    0, "top right y", 0,  0, 5610)

exit(gen.generate(PACKAGE, "no_entry_layer", "carto_mask_config"))